Makefile

```diff
@@ -40,12 +40,12 @@ test: checkall testall
 
 testfast:
 	@echo 'TESTSUITE'
-	cargo test --features=with_test_support
+	cd sentry && cargo test --features=with_test_support
```

Comment:
Can you use `cargo test -p sentry` here, instead? It would be nice to just run `cargo test` and have all workspace members tested, rather than selecting a package, because otherwise it'll be hard to grow this list later.